app/assets/stylesheets/peer_reviews.css

```diff
@@ -0,0 +1 @@
+@import 'assignments';
```

Comment:
Remove this file as well (doesn't come in yet). Though do remember this change for later.